docs/examples/sock-shop.md

```diff
@@ -4,52 +4,83 @@ sidebar_position: 8
 
-To deploy the *Sock Shop*, we need to use `gitops add app`. `gitops add app` comes in three flavors depending on how you'd like your GitOps support to be managed. Just as GitOps in general lets you synchronize your application definitions with their in-cluster versions via `git` operations and pull requests, Weave GitOps lets you manage the resources that _do_ the synchronization in the same way. That might sound a bit "meta" but it allows you to update the management resources themselves via pull requests. You can change things like synchronization intervals or even upgrade the resources via git. Why _wouldn't_ you want to be able to do those things?
-
-The three storage options for the management resources provided by Weave GitOps are:
-- in a `.wego` subdirectory of the same repository as your application (for a simple all-in-one configuration)
-- in a separate configuration repository (allowing you to collect all your management resources together)
-- _only_ in the cluster (this is mostly intended for quick turnaround and testing)
+To deploy the *Sock Shop*, we need to use `gitops add app`. `gitops add app` will store the GitOps automation support for your application in the `.weave-gitops` directory of the configuration repository you specified at install time. The definition of your application can be stored either in a separate repository or in the configuration repository itself (for a simple all-in-one configuration). If you want to store the application resources in the configuration repository, you only need to specify the `--url` flag which will be used for both application and configuration resources; however, this assumes that the application repository URL was passed to `gitops install`. If you want the application resources to be stored separately, you need to specify both `--url` and `--app-config-url` parameters. The `--url` parameter should be the URL of the repository containing the application definition and the `--app-config-url` parameter should be the URL that was used in `gitops install`.
 
 First, let's fork the *Sock Shop* repository. You can simply go to the [repository](https://github.com/microservices-demo/microservices-demo) in `GitHub` and select `Fork`.
 
-Now, if we just want to get the *Sock Shop* running in the simplest way possible, without modifying anything, we can run a single command:
+Now, we can get the *Sock Shop* running as simply as possible using a separate configuration repository with:
 
 ```console
-> gitops add app --url ssh://git@github.com/example/microservices-demo.git --path ./deploy/kubernetes/manifests --app-config-url NONE
-◎ Checking cluster status
-✔ GitOps installed
-uploading deploy key
-Deploy key generated and uploaded to git provider
+> gitops add app --url ssh://git@github.com/example/microservices-demo.git --path ./deploy/kubernetes/manifests --app-config-url ssh://git@github.com/example/external.git --auto-merge
```

Comment:
Suggest breaking the command out into a separate console block to allow the user to more easily copy/paste and distinguish between cmd and output. Not essential for merge.